packages/eas-cli/src/utils/expoCli.ts

```diff
@@ -4,7 +4,11 @@ import resolveFrom from 'resolve-from';
 
 import Log from '../log';
 
-export async function expoCommandAsync(projectDir: string, args: string[]): Promise<void> {
+export async function expoCommandAsync(
+  projectDir: string,
+  args: string[],
+  silent?: boolean
```

Comment:
```suggestion
  { silent = false } = { silent?: boolean }
```
So now the call-site can look better (you don't need a comment):
```
await expoCommandAsync(
    projectDir,
    ['export', '--output-dir', inputDir, '--experimental-bundle'],
    { silent: true }
  );
```